.github/workflows/build.yaml

```diff
@@ -17,90 +17,47 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu-latest ]
-        arch: [ amd64 ]
-        cmake-arch: [ x86_64 ]
-        openssl-arch: [ linux-x86_64 ]
         include:
+          # TODO: Linking is not working on linux.
+#          - os: ubuntu-latest
+#            cmake-arch: x86_64
+#            openssl-arch: linux-x86_64
+#            sqlite-arch: x64-gnu-linux
+#            task: linkReleaseExecutableLinuxX64
           - os: macOS-latest
-            arch: x86_64
             cmake-arch: x86_64
             openssl-arch: darwin64-x86_64-cc
-          - os: macOS-latest
-            arch: aarch64
-            cmake-arch: arm64
-            openssl-arch: darwin64-arm64-cc
+            sqlite-arch: x64-apple-macos
+            task: linkReleaseExecutableMacosX64
+          # TODO: Cross-compilation is not working. There's so many dependencies that need to be manually built.
+#          - os: macOS-latest
+#            cmake-arch: arm64
+#            openssl-arch: darwin64-arm64-cc
+#            sqlite-arch: arm64-apple-macos
+#            task: linkReleaseExecutableMacosArm64
           # TODO: build on 'windows-latest'
 
     runs-on: ${{ matrix.os }}
 
-    steps:
-      - uses: actions/checkout@v4
-
-      - name: Install dependencies
-        if: matrix.os == 'macOS-latest'
-        run: brew install coreutils
-
-      - name: Build native library
-        run: ./.github/workflows/build-deps.sh -a ${{ matrix.arch }} -c ${{ matrix.cmake-arch }} -o ${{ matrix.openssl-arch }}
-
-      - name: Upload native libraries
-        uses: actions/upload-artifact@v3
-        with:
-          name: native-libraries
-          path: ./native/*
-          if-no-files-found: error
-
-      - name: Upload libgit2 includes
-        if: matrix.os == 'ubuntu-latest'
-        uses: actions/upload-artifact@v3
-        with:
-          name: libgit-include
-          path: ./libgit2/include
-          if-no-files-found: error
-
-  build:
-    needs: build_deps
-
-    strategy:
-      fail-fast: false
-      matrix:
-        os: [ ubuntu-latest, macOS-latest ]
-
-    runs-on: ${{ matrix.os }}
-
     steps:
       - uses: actions/checkout@v4
       - uses: actions/setup-java@v3
         with:
           distribution: 'zulu'
           java-version: 21
 
-      - name: Download native libraries
-        uses: actions/download-artifact@master
-        with:
-          name: native-libraries
-          path: native
-
-      - name: Download libgit2 includes
-        uses: actions/download-artifact@master
-        with:
-          name: libgit-include
-          path: libgit2/include
+      - name: Install dependencies
+        if: matrix.os == 'macOS-latest'
+        run: brew install coreutils
```

Comment:
This was for objdump that jextract needed, right? Can probably be dropped.

Reply:
Uhh, I thought it was for `make`... I can try without.